Tests/Functional/ContainerTest.php

```diff
@@ -92,6 +92,16 @@ public static function imageScalingValuesDataProvider(): iterable
                 '4' => 'large 748.584 (min-width: 1480px)',
             ],
         ];
+        yield '2 Column in 1 Column' => [
```

Comment:
There are two columns with the same picture in it. Both should have the same size. The test will pass if only one image was calculated correctly or different assertions will pass through different images.
The chance is probably very small.

Reply:
Yes - you're right. I will change that.